.github/workflows/vulnerability_scanning.yml

```diff
@@ -2,45 +2,27 @@ name: Workflow to scan vulnerability
 on:
   workflow_call:
     inputs:
-      image_tar_name:
+      docker_image_fullname:
+          type: string
+          required: true
+      build_runner_name:
         type: string
-        required: false
+        required: true
       runner:
         ## ['cn', 'us', 'cn1', 'cn2', 'cn3', 'us1', 'us2', 'us3', ' ubuntu-latest']
         type: string
-        ## use github hosted runner by default
+        # ## use github hosted runner by default
         default: "ubuntu-latest"
         required: false
-    secrets:
-      MINIO_ACCESS_KEY_ID:
-        required: false
-      MINIO_SECRET_ACCESS_KEY:
-        required: false
-      MINIO_ENDPOINT_URL:
-        required: false
+
+
 jobs:
   scanning:
     name: vulnerability scanning
-    runs-on: ${{ inputs.runner }}
+    runs-on: [ self-hosted, "${{ inputs.build_runner_name }}" ]
```

Comment:
any reason to not use `runner`?

Reply:
since many places use this name, to make a different name for distinguish.if you think it's ok, I will update.